[Workspace Chrome] Visual Improvements

tsullivan opened the pull request:
## Summary

Update the look and feel of the Solution chrome.

Epic: https://github.com/elastic/kibana-team/issues/2106

Includes https://github.com/elastic/kibana/pull/245099

**Changes:**
| Area | Change
|-|-
| Navigation collapse button | <li>moved from the chrome header to the navigation footer
| Application space, App menu bar | <li>globally consistent white background (due to changes in EUI v110)<br><li>drop shadow
| Application space| <li>Spacing at the bottom and right sides between the app content<br>Spacing when the navigation side panel shows (conditionally)
| Global header background | <li>shaded

### Things to test
1. dark mode
2. the expanded/collapsed nav
3. secondary nav panel
4. embedded console button at the bottom of the screen (Elasticsearch space)
5. high contrast mode

## Screenshots
**Before**

1. <img width="1835" height="1101" alt="image" src="https://github.com/user-attachments/assets/75a1ded1-f102-42ec-bc7e-804c27b4455b" />
6. <img width="1835" height="1101" alt="image" src="https://github.com/user-attachments/assets/5301dd03-4cdb-4cd3-8fba-517adfd26b29" />
7. <img width="1835" height="1101" alt="image" src="https://github.com/user-attachments/assets/b69c176b-7a8a-4d1f-879f-970010381483" />

**After**
1. <img width="1836" height="1100" alt="image" src="https://github.com/user-attachments/assets/7d2b9d75-ee83-4893-83f4-9a6561fd1e04" />
2. <img width="1835" height="1100" alt="image" src="https://github.com/user-attachments/assets/ae70825b-a52c-43b5-867f-3efd02681c60" />
3. <img width="1835" height="1100" alt="image" src="https://github.com/user-attachments/assets/177d896c-8e9b-4810-839c-934ea9c7a3a8" />

### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [x] Audit all Kibana pages for consistency of stylistic changes
- ~~[ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)~~
- ~~[ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials~~
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- ~~[ ] If a plugin configuration key changed, check if it needs to be allowlisted in the cloud and added to the [docker list](https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker)~~
- ~~[ ] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.~~
- ~~[ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed~~
- ~~[ ] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)~~
- ~~[ ] Review the [backport guidelines](https://docs.google.com/document/d/1VyN5k91e5OVumlc0Gb9RPa3h1ewuPE705nRtioPiTvY/edit?usp=sharing) and apply applicable `backport:*` labels.~~

### Identify risks

~~Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.~~

~~Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.~~

- ~~[ ] [See some risk examples](https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx)~~
- ~~[ ] ...~~

tsullivan committed:
Make header background transparent

tsullivan committed:
Make sidenav background transparent

tsullivan committed:
Make appbar background white

tsullivan committed:
Add support for marginRight and marginBottom to layout container to enable this "app sheet" look

tsullivan committed:
Add border-radius and shadow to app container

tsullivan committed:
Adjust header logos and collapse button sizing and spacing (WIP)

tsullivan committed:
Install custom EUI v109 build no.0001 for page template style testing

tsullivan committed:
Remove app space background color overrides

tsullivan committed:
Fix for making sidenav background transparent

tsullivan committed:
Replace hardcoded px value with EUI token

tsullivan committed:
Add new side nav dividers

tsullivan committed:
update jest snapshots

tsullivan committed:
Remove Fleet background style customizations

tsullivan committed:
Install custom EUI v109 build no.0002 for page template style testing

weronikaolejniczak committed:
chore(eui): use noItemsMessage prop in EuiInMemoryTable

Commit:
Update test to reflect role change of search input from "searchbox" to "combobox" in DataView list component

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-i

tsullivan committed:
Install custom EUI build of main branch no.003

tsullivan committed:
Fix nav borders

tsullivan committed:
Revert "update jest snapshots"

This reverts commit 5b7efdc8d69d163c354d11884d4c7b68a26a6942.

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-i

Dosant committed:
[Chrome Grid] Revert the header displayed next to the sidenav (#244628)

One user-facing change we made with [the grid
layout](https://github.com/elastic/kibana/pull/243181/) is that we
started displaying the Kibana header next to the sidenav instead of
above it.

**Before grid layout**

<img width="630" height="409" alt="Screenshot 2025-10-31 at 11 22 32"
src="https://github.com/user-attachments/assets/f04eaf6a-9605-4346-b105-e172b5f990f2"
/>

**With grid layout**

<img width="591" height="290" alt="Screenshot 2025-10-31 at 11 25 24"
src="https://github.com/user-attachments/assets/6e34f379-d979-4c82-a6fc-1c94811f17b6"
/>

-----

With this PR, we revert this user-facing change and return the header to
the top of the nav.

<img width="1624" height="1056" alt="Screenshot 2025-11-28 at 12 54 38"
src="https://github.com/user-attachments/assets/b8bb6a56-4983-4341-9ae6-e52ac46b5636"
/>

tsullivan committed:
Merge commit '4fc8027~1' into workspace-chrome/visual-improvements-i

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-i

tsullivan committed:
use proper border for app space

The layout wrapper has a floating border on a pseudo element, but the children have different z-index levels (e.g. the top bar) which then overlap the container border. Since we want the border to be visible at all times, a regular border would be more fitting here.

Co-authored-by: Lene Gadewoll <lene.gadewoll@elastic.co>

mgadewoll committed:
chore: update eui packages

mgadewoll committed:
refactor: update message prop

mgadewoll committed:
refactor: remove obsolete color prop

mgadewoll committed:
refactor: check for available query arg

- EUI updated the type and the query might be null for text searches

mgadewoll committed:
test: use noItemsMessage prop

mgadewoll committed:
test: update snapshots

ek-so committed:
Changed the buttons

mgadewoll committed:
Merge branch 'main' into eui/v110.0.0

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-i

mgadewoll committed:
Merge branch 'main' into eui/v110.0.0

ek-so committed:
Visual tweaks for the feedback snippet

ek-so committed:
Changed feedback button style

ek-so committed:
Centered the thumb up animation back

tsullivan committed:
Merge branch 'eui/v110.0.0' into workspace-chrome/visual-improvements-i

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-i

tsullivan committed:
package.json newline change

tsullivan committed:
Plain background style for secondary menu

tsullivan committed:
Plain background style for secondary menu

tsullivan committed:
Border and shadow edges for side panel

tsullivan committed:
Move collapse button to side nav footer

tsullivan committed:
Add line above collapse button

tsullivan committed:
clean up work to use test EUI build

tsullivan committed:
clean up hardcoded px

tsullivan committed:
fix panel title background style

tsullivan committed:
clean up empty header spinner style

tsullivan committed:
fix nav padding-bottom

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-i

tsullivan committed:
pass `isSidePanelOpen` with the navigation `setWidth` calls to allow the layout to add panel-specific gap spacing

tsullivan committed:
change the package responsible for side panel gap to the navigation package

tsullivan committed:
re-fix the panel title background style

tsullivan committed:
reset yarn.lock

tsullivan committed:
fix side panel bottom margin

tsullivan committed:
layout constant for side panel content gap

tsullivan committed:
move collapse button to navigation package

tsullivan committed:
fix remnant left divider on header top bar

tsullivan committed:
fix alignment of elastic logo (header) and solution logo (collapsed nav)

tsullivan committed:
fix ts leftover from previous solution

tsullivan committed:
clean up remnants from previous changes to nav API

tsullivan committed:
refactor: centralize navigation collapse callback and improve documentation

- Move onToggleCollapsed callback into navProps in chrome_service.tsx
- Make onToggleCollapsed a required prop instead of optional
- Add comprehensive documentation explaining architectural decision:
  - State managed in chrome_service as BehaviorSubject with localStorage persistence
  - External consumers: feedback button visibility and public chrome API
- Remove redundant toggle prop drilling from layout components
- Update test component to provide required callback
- Simplify component interfaces by removing duplicate props

kibanamachine committed:
Changes from node scripts/eslint_all_files --no-cache --fix

elasticmachine commented:
Pinging @elastic/fleet (Team:Fleet)

nchaulet reviewed (approved):
Fleet changes LGTM 🚀

ek-so committed:
Merge branch 'elastic:main' into ekso-feedback-ui-tweaks

ek-so committed:
Changed FB button back to empty

ek-so committed:
Merge branch 'main' into ekso-feedback-ui-tweaks

kibanamachine committed:
Changes from node scripts/eslint_all_files --no-cache --fix

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-ii

tkajtoch commented:
@tsullivan Do we have designs for the chrome updates in high contrast mode? It mostly looks correct but I feel like some added borders around the new chrome "panels" would make a lot of sense there. I think the shadows make it pop and elevate it above, while in high contrast mode it's harder to distinguish different content areas

<img width="1728" height="899" alt="Screenshot 2025-12-11 at 16 07 58" src="https://github.com/user-attachments/assets/9f8112bb-1465-4d1f-b730-5da2e03ae03a" />

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/371880)
* Commit: 2730003f90ffd19994ac6fd0907d87c63883372b
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-245898)

### Failed CI Steps
* [Jest Tests #16](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0e04-22e9-4bad-9062-815046df5147)
* [Osquery Cypress Tests on Serverless #1](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0ddb-d9f6-4775-9fdf-c8f76ab04dba)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0ddb-d9f6-4775-9fdf-c8f76ab04dba) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/371880/jobs/019b0ddb-d9f6-4775-9fdf-c8f76ab04dba/artifacts/019b0e15-e583-45f1-912c-2480b8c3d0f5) Osquery Cypress Tests on Serverless #<span></span>1 / ALL - Saved queries should create a new query and verify: 
 - hidden columns, full screen and sorting 
- pagination 
- query can viewed (status), edited and deleted 
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0e04-22e9-4bad-9062-815046df5147) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/371880/jobs/019b0e04-22e9-4bad-9062-815046df5147/artifacts/019b0e14-b77e-4694-bfe3-e8fa6538a9e6) Jest Tests #<span></span>16 / Both modes should render the side navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0e04-22e9-4bad-9062-815046df5147) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/371880/jobs/019b0e04-22e9-4bad-9062-815046df5147/artifacts/019b0e14-b77f-442c-b752-6a4981db8d1b) Jest Tests #<span></span>16 / Collapsed mode should render the side navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0e04-22e9-4bad-9062-815046df5147) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/371880/jobs/019b0e04-22e9-4bad-9062-815046df5147/artifacts/019b0e14-b77f-4068-bbd7-75fba8bc0877) Jest Tests #<span></span>16 / Expanded mode should render the side navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/371880#019b0e04-22e9-4bad-9062-815046df5147) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/371880/jobs/019b0e04-22e9-4bad-9062-815046df5147/artifacts/019b0e14-b780-4727-b30e-7c1dc6fb96d5) Jest Tests #<span></span>16 / useLayoutWidth sets the collapsed width when the navigation is collapsed

### History
* :broken_heart: [Build #371459](https://buildkite.com/elastic/kibana-pull-request/builds/371459) failed 72ff0ea02222b3a8d8102af97135fd7772097e18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371880","number":371880,"commit":"2730003f90ffd19994ac6fd0907d87c63883372b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371459","number":371459,"commit":"72ff0ea02222b3a8d8102af97135fd7772097e18"}],"number":371880}
buildkite-pr-comment-->

tsullivan committed:
increase SIDE_PANEL_WIDTH +8

tsullivan committed:
fix sizing of collapse button and bottom-padding of nav wrapper

tsullivan committed:
adjust width of nav dividers

tsullivan committed:
use pseudo-divider for nav side panel sections

tkajtoch reviewed (changes requested):
Code changes look great; I tested all changes locally and found no issues other than these two:

* Nit, likely unrelated to the changes from this PR: On small screens, the collapse/expand side nav button is hidden but still renders the separator line / border.
* The high contrast mode looks off and to me feels like a regression from what we had before. I think we just need a few more styles and it'll be good to go, but we'd need designer input on that. See the comment above for more details.

I'm going to mark this as _Request changes_ due to the high contrast mode, but overall, the changes made here are solid and everything else looks sharp! Awesome job!

tsullivan committed:
fixes for high contrast mode

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/372112)
* Commit: 42188ef524d1a94c601ad5fc5be48238c6ab6b7a
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-245898)

### Failed CI Steps
* [Osquery Cypress Tests on Serverless #1](https://buildkite.com/elastic/kibana-pull-request/builds/372112#019b0f3c-2901-48c6-8aa7-65806c9f5792)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372112#019b0f3c-2901-48c6-8aa7-65806c9f5792) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372112/jobs/019b0f3c-2901-48c6-8aa7-65806c9f5792/artifacts/019b0f75-07c5-4f2d-85c3-56bffee2abb4) Osquery Cypress Tests on Serverless #<span></span>1 / ALL - Saved queries should create a new query and verify: 
 - hidden columns, full screen and sorting 
- pagination 
- query can viewed (status), edited and deleted 

### History
* :broken_heart: [Build #371880](https://buildkite.com/elastic/kibana-pull-request/builds/371880) failed 2730003f90ffd19994ac6fd0907d87c63883372b
* :broken_heart: [Build #371459](https://buildkite.com/elastic/kibana-pull-request/builds/371459) failed 72ff0ea02222b3a8d8102af97135fd7772097e18

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372112","number":372112,"commit":"42188ef524d1a94c601ad5fc5be48238c6ab6b7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371880","number":371880,"commit":"2730003f90ffd19994ac6fd0907d87c63883372b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371459","number":371459,"commit":"72ff0ea02222b3a8d8102af97135fd7772097e18"}],"number":372112}
buildkite-pr-comment-->

tsullivan committed:
fix collapse button divider hiding for small screen

tsullivan commented:
@tkajtoch I implemented fixes in response to [your feedback](https://github.com/elastic/kibana/pull/245898#pullrequestreview-3569195468):

- fixes for high contrast mode 0da4edf
- fix collapse button divider hiding for small screen c939a8d

tsullivan committed:
fix jest unit tests

kibanamachine committed:
Changes from node scripts/eslint_all_files --no-cache --fix

ek-so committed:
jest tests

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-ii

tsullivan committed:
Merge branch 'ekso-feedback-ui-tweaks' into workspace-chrome/visual-improvements-ii

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/372477)
* Commit: f660e8165e6690a261b4692d7072d0c52d7f1aac
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-245898)

### Failed CI Steps
* [FTR Configs #1](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f147-4869-980a-b64998c3a068)
* [FTR Configs #1](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b1348-7127-46ad-a395-8cfe1b132347)
* [FTR Configs #19](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f15b-4d8a-9732-3edd909168cf)
* [FTR Configs #42](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f174-4326-9eb0-694a698c174f)
* [FTR Configs #52](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f17c-4ab4-a097-0e6b14017681)
* [FTR Configs #97](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f1a6-4922-bd6d-255e0c8af000)
* [FTR Configs #127](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f1ca-4395-a264-fe7dae6da436)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f15b-4d8a-9732-3edd909168cf) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b132d-f15b-4d8a-9732-3edd909168cf/artifacts/019b1352-de7d-4e14-954a-a7bae82f3b39) FTR Configs #<span></span>19 / discover/group1 discover histogram should modify the time range when the histogram is brushed
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f17c-4ab4-a097-0e6b14017681) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b132d-f17c-4ab4-a097-0e6b14017681/artifacts/019b1349-9bba-4917-921e-4d80c47b8b70) FTR Configs #<span></span>52 / discover/group1 discover histogram should modify the time range when the histogram is brushed
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f1ca-4395-a264-fe7dae6da436) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b132d-f1ca-4395-a264-fe7dae6da436/artifacts/019b1351-9d72-4769-8751-855df225eafc) FTR Configs #<span></span>127 / discover/group1 discover histogram should modify the time range when the histogram is brushed
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f147-4869-980a-b64998c3a068) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b132d-f147-4869-980a-b64998c3a068/artifacts/019b1348-37e4-4c9b-bf36-e277874f0543) FTR Configs #<span></span>1 / serverless observability UI navigation renders a feedback callout
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b1348-7127-46ad-a395-8cfe1b132347) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b1348-7127-46ad-a395-8cfe1b132347/artifacts/019b1351-d0d7-4bba-9c39-7ff89d7695af) FTR Configs #<span></span>1 / serverless observability UI navigation renders a feedback callout
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f1a6-4922-bd6d-255e0c8af000) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b132d-f1a6-4922-bd6d-255e0c8af000/artifacts/019b1353-2384-481c-9e18-b4d9cfeba399) FTR Configs #<span></span>97 / serverless security UI navigation shows cases in sidebar navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372477#019b132d-f174-4326-9eb0-694a698c174f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372477/jobs/019b132d-f174-4326-9eb0-694a698c174f/artifacts/019b1354-c3a5-4142-9705-83b1d78fbf37) FTR Configs #<span></span>42 / Solution navigation smoke tests observability solution sidenav & breadcrumbs renders the correct nav and navigate to links

### History
* :broken_heart: [Build #372158](https://buildkite.com/elastic/kibana-pull-request/builds/372158) failed 1ba7cd55a9786815c6261ef6f2e3809b67c0504e
* :broken_heart: [Build #372151](https://buildkite.com/elastic/kibana-pull-request/builds/372151) failed 81174c6ed47642c695187d04b7db2fee08a48e74
* :broken_heart: [Build #372112](https://buildkite.com/elastic/kibana-pull-request/builds/372112) failed 42188ef524d1a94c601ad5fc5be48238c6ab6b7a
* :broken_heart: [Build #371880](https://buildkite.com/elastic/kibana-pull-request/builds/371880) failed 2730003f90ffd19994ac6fd0907d87c63883372b

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372477","number":372477,"commit":"f660e8165e6690a261b4692d7072d0c52d7f1aac"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372158","number":372158,"commit":"1ba7cd55a9786815c6261ef6f2e3809b67c0504e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372151","number":372151,"commit":"81174c6ed47642c695187d04b7db2fee08a48e74"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372112","number":372112,"commit":"42188ef524d1a94c601ad5fc5be48238c6ab6b7a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/371880","number":371880,"commit":"2730003f90ffd19994ac6fd0907d87c63883372b"}],"number":372477}
buildkite-pr-comment-->

tkajtoch reviewed (approved):
I reviewed and tested the high contrast mode updates and the collapse button change. Thank you for addressing my feedback! LGTM

tsullivan committed:
update baselines for ui visual regression testing

tsullivan committed:
fix functional test expectation impacted by visual changes

tsullivan committed:
fix functional test for rendering the correct nav and using the nav links

These tests began to fail because the navigation menu items that used to
be clickable now need to be accessed by first clicking the "More" button
in the navigation.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/372732)
* Commit: c0ea163c9d5762fe5320882e7fa7662f9cb20f84
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-245898)

### Failed CI Steps
* [FTR Configs #59](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-835e-4886-9d67-e56edc2c3f43)
* [FTR Configs #59](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b1490-fba4-4aae-a3b4-6b715c3b4e4c)
* [FTR Configs #82](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-8376-403a-ae90-99b6164c7dd3)
* [FTR Configs #99](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-8386-4ec0-887f-52c4b819044d)
* [FTR Configs #99](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b148c-1f99-4562-a9d1-5b75ad38b0c1)
* [FTR Configs #119](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-839f-44d0-919e-d915feb04108)
* [FTR Configs #119](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b1489-c682-4ea8-8f54-0c4309ed7b3e)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-8376-403a-ae90-99b6164c7dd3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b146e-8376-403a-ae90-99b6164c7dd3/artifacts/019b1497-1b21-4764-9ddd-38b80250eb9e) FTR Configs #<span></span>82 / Endpoint Policy @<span></span>ess @<span></span>serverless Creation of DOT indices for elastic defend policies "before each" hook for "should create indices when endpoint integration policy is created"
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-839f-44d0-919e-d915feb04108) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b146e-839f-44d0-919e-d915feb04108/artifacts/019b1489-a6cb-4f91-9207-e3e9637593d3) FTR Configs #<span></span>119 / serverless observability UI navigation renders a feedback callout
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b1489-c682-4ea8-8f54-0c4309ed7b3e) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b1489-c682-4ea8-8f54-0c4309ed7b3e/artifacts/019b1493-4075-4a8f-acbe-1f3e4b83245b) FTR Configs #<span></span>119 / serverless observability UI navigation renders a feedback callout
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-8386-4ec0-887f-52c4b819044d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b146e-8386-4ec0-887f-52c4b819044d/artifacts/019b148b-fd2b-45e5-bb38-217869bfeeab) FTR Configs #<span></span>99 / serverless security UI navigation shows cases in sidebar navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b148c-1f99-4562-a9d1-5b75ad38b0c1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b148c-1f99-4562-a9d1-5b75ad38b0c1/artifacts/019b1495-3550-4520-81e5-16defe8c3b87) FTR Configs #<span></span>99 / serverless security UI navigation shows cases in sidebar navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b146e-835e-4886-9d67-e56edc2c3f43) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b146e-835e-4886-9d67-e56edc2c3f43/artifacts/019b1490-c9a6-4237-aff6-b3b4b67b5ffa) FTR Configs #<span></span>59 / Solution navigation smoke tests observability solution sidenav & breadcrumbs renders the correct nav and navigate to links
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/372732#019b1490-fba4-4aae-a3b4-6b715c3b4e4c) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/372732/jobs/019b1490-fba4-4aae-a3b4-6b715c3b4e4c/artifacts/019b1496-d466-4a11-8c0d-7f164b999cda) FTR Configs #<span></span>59 / Solution navigation smoke tests observability solution sidenav & breadcrumbs renders the correct nav and navigate to links

### History
* :broken_heart: [Build #372477](https://buildkite.com/elastic/kibana-pull-request/builds/372477) failed f660e8165e6690a261b4692d7072d0c52d7f1aac
* :broken_heart: [Build #372158](https://buildkite.com/elastic/kibana-pull-request/builds/372158) failed 1ba7cd55a9786815c6261ef6f2e3809b67c0504e
* :broken_heart: [Build #372151](https://buildkite.com/elastic/kibana-pull-request/builds/372151) failed 81174c6ed47642c695187d04b7db2fee08a48e74
* :broken_heart: [Build #372112](https://buildkite.com/elastic/kibana-pull-request/builds/372112) failed 42188ef524d1a94c601ad5fc5be48238c6ab6b7a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372732","number":372732,"commit":"c0ea163c9d5762fe5320882e7fa7662f9cb20f84"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372477","number":372477,"commit":"f660e8165e6690a261b4692d7072d0c52d7f1aac"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372158","number":372158,"commit":"1ba7cd55a9786815c6261ef6f2e3809b67c0504e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372151","number":372151,"commit":"81174c6ed47642c695187d04b7db2fee08a48e74"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372112","number":372112,"commit":"42188ef524d1a94c601ad5fc5be48238c6ab6b7a"}],"number":372732}
buildkite-pr-comment-->

tsullivan committed:
fix for functional test feedback snippet changes

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-ii

kibanamachine committed:
Changes from node scripts/eslint_all_files --no-cache --fix

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/372754)
* Commit: a4fb4dccca44da47859782d6e4e5406d7ab4b5eb

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/372754#019b14af-bbe2-4d56-9fa7-a54930c7579f)

### History
* :broken_heart: [Build #372732](https://buildkite.com/elastic/kibana-pull-request/builds/372732) failed c0ea163c9d5762fe5320882e7fa7662f9cb20f84
* :broken_heart: [Build #372477](https://buildkite.com/elastic/kibana-pull-request/builds/372477) failed f660e8165e6690a261b4692d7072d0c52d7f1aac
* :broken_heart: [Build #372158](https://buildkite.com/elastic/kibana-pull-request/builds/372158) failed 1ba7cd55a9786815c6261ef6f2e3809b67c0504e
* :broken_heart: [Build #372151](https://buildkite.com/elastic/kibana-pull-request/builds/372151) failed 81174c6ed47642c695187d04b7db2fee08a48e74
* :broken_heart: [Build #372112](https://buildkite.com/elastic/kibana-pull-request/builds/372112) failed 42188ef524d1a94c601ad5fc5be48238c6ab6b7a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372754","number":372754,"commit":"a4fb4dccca44da47859782d6e4e5406d7ab4b5eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372732","number":372732,"commit":"c0ea163c9d5762fe5320882e7fa7662f9cb20f84"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372477","number":372477,"commit":"f660e8165e6690a261b4692d7072d0c52d7f1aac"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372158","number":372158,"commit":"1ba7cd55a9786815c6261ef6f2e3809b67c0504e"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372151","number":372151,"commit":"81174c6ed47642c695187d04b7db2fee08a48e74"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372112","number":372112,"commit":"42188ef524d1a94c601ad5fc5be48238c6ab6b7a"}],"number":372754}
buildkite-pr-comment-->

clintandrewhall commented on src/core/packages/chrome/layout/core-chrome-layout-components/layout_global_css.tsx:
```diff
@@ -80,10 +82,17 @@ export const LayoutGlobalCSS = () => {
   `;
 
   const application = css`
+    ${layoutVarName('application.marginBottom')}: ${applicationMarginBottom}px;
```
nit: should we go ahead and add margin top and left as 0, for completeness and flexibility?

clintandrewhall commented on src/core/packages/chrome/layout/core-chrome-layout-constants/src/css_variables.ts:
```diff
@@ -15,7 +15,15 @@ export type LayoutComponent =
   | 'application';
 export type ApplicationComponent = 'topBar' | 'bottomBar' | 'content';
-export type LayoutProperty = 'top' | 'bottom' | 'left' | 'right' | 'height' | 'width';
+export type LayoutProperty =
```
nit: can this be derived, rather than explicitly stated?  It's probably fine, given the other properties are enumerated statically, as well, but just curious if a `Pick<>` could make it more clear where we expect these to come from, (e.g. styles)

tsullivan replied:
5c61c0cf9c00

clintandrewhall commented on src/core/packages/chrome/navigation/src/__tests__/test_component.tsx:
```diff
@@ -28,6 +28,7 @@ export const TestComponent = ({
   logo,
 }: TestComponentProps) => {
   const [activeItemId, setActiveItemId] = useState(initialActiveItemId);
+  const [collapsed, setCollapsed] = useState(isCollapsed);
```
```suggestion
  const [isCollapsed, setIsCollapsed] = useState(isCollapsedProp);
```

clintandrewhall commented on src/core/packages/chrome/navigation/src/__tests__/test_component.tsx:
```diff
@@ -39,10 +40,11 @@ export const TestComponent = ({
       <Navigation
         activeItemId={activeItemId}
-        isCollapsed={isCollapsed}
+        isCollapsed={collapsed}
```
```suggestion
        isCollapsed={isCollapsed}
```

clintandrewhall commented on src/core/packages/chrome/navigation/src/components/side_nav/logo.tsx:
```diff
@@ -48,10 +49,26 @@ export const Logo = ({
     padding-top: ${isCollapsed ? euiTheme.size.s : euiTheme.size.m};
     padding-bottom: ${isCollapsed ? '7px' : euiTheme.size.m};
 
+    ${highContrastMode
```
This _feels_ repeated in a number of places.  Any chance to DRY this out?

clintandrewhall commented on src/core/packages/chrome/navigation/src/hooks/use_layout_width.ts:
```diff
@@ -11,7 +11,7 @@ import { useEffect } from 'react';
 export const COLLAPSED_WIDTH = 48;
 export const EXPANDED_WIDTH = 100;
-export const SIDE_PANEL_WIDTH = 240;
+export const SIDE_PANEL_WIDTH = 248;
```
nit: consider a note explaining this value, such as the offset.

clintandrewhall reviewed (approved):
Great work!  Just some nits from my end.

Dosant commented on src/core/packages/chrome/layout/core-chrome-layout/layouts/grid/grid_global_app_style.tsx:
```diff
@@ -9,77 +9,91 @@
+      top: var(--kbnHeaderBannerHeight);
+    }
+
+    .kbnChromeLayoutApplication {
```
curious why are we doing it here instead of application component itself?

tsullivan replied:
@Dosant just a head's up: my hasty change in response caused a bug that @ryankeairns is fixing in https://github.com/elastic/kibana/pull/248543

Dosant commented on src/core/packages/chrome/layout/core-chrome-layout-components/layout_global_css.tsx:
```diff
@@ -80,10 +82,17 @@ export const LayoutGlobalCSS = () => {
+    ${layoutVarName('application.marginRight')}: ${applicationMarginRight}px;
+
     ${layoutVarName('application.top')}: ${bannerHeight + headerHeight}px;
-    ${layoutVarName('application.bottom')}: ${layoutVar('footer.height')};
```
TODO for myself: we should refactor these variables to use static calculation as much as possible instead of dynamic `calc` for runtime perf reasons

Dosant commented on src/core/packages/chrome/layout/core-chrome-layout-components/application/layout_application.styles.ts:
```diff
@@ -9,21 +9,33 @@
-    position: relative;
-    width: 100%;
+
+    height: calc(100% - ${layoutVar('application.marginBottom')});
```
I think the original CSS might have been redundant, and explicit height and width might not be needed here at all.

I'll create an issue for myself to double check this if can be simplified

ThomThomson reviewed (approved):
Dashboard screenshot baseline updates LGTM

tsullivan committed:
Merge branch 'main' into workspace-chrome/visual-improvements-ii

tsullivan committed:
hide scrollbar in application grid cell

justinkambic reviewed (approved):
Obs UI change looks good to me!

tsullivan committed:
use derived values for the LayoutProperty type

tsullivan committed:
state variables rename

mohamedhamed-ahmed reviewed (approved):
LGTM!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/378044)
* Commit: 9a6283e1ff0fdcc5e6f8a324da732dfac9024442
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-245898)

### Failed CI Steps
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f7f-ece4-464f-a2a2-0e259384731d)
* [Jest Tests #7](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f8d-bb10-4627-86ac-805012fa4907)
* [FTR Configs #120](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f7f-ed9b-40e4-be3d-2b41b2b48d4c)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f7f-ece4-464f-a2a2-0e259384731d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/378044/jobs/019b8f7f-ece4-464f-a2a2-0e259384731d/artifacts/019b8f8d-9cd9-4786-8c05-ec9335d80e0d) Jest Tests #<span></span>7 / Both modes should render the side navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f8d-bb10-4627-86ac-805012fa4907) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/378044/jobs/019b8f8d-bb10-4627-86ac-805012fa4907/artifacts/019b8f9b-abcf-4d72-88e9-9f777e5f6d29) Jest Tests #<span></span>7 / Both modes should render the side navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f7f-ece4-464f-a2a2-0e259384731d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/378044/jobs/019b8f7f-ece4-464f-a2a2-0e259384731d/artifacts/019b8f8d-9cd8-48b3-9da3-d0da59c3182f) Jest Tests #<span></span>7 / Expanded mode should render the side navigation
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/378044#019b8f8d-bb10-4627-86ac-805012fa4907) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/378044/jobs/019b8f8d-bb10-4627-86ac-805012fa4907/artifacts/019b8f9b-abcf-4208-8f23-0619785a669a) Jest Tests #<span></span>7 / Expanded mode should render the side navigation

### History
* :broken_heart: [Build #372761](https://buildkite.com/elastic/kibana-pull-request/builds/372761) failed 9a5d3c0f9ef99db9184528ca1e1baebb60b0bc60
* :broken_heart: [Build #372754](https://buildkite.com/elastic/kibana-pull-request/builds/372754) failed a4fb4dccca44da47859782d6e4e5406d7ab4b5eb
* :broken_heart: [Build #372732](https://buildkite.com/elastic/kibana-pull-request/builds/372732) failed c0ea163c9d5762fe5320882e7fa7662f9cb20f84
* :broken_heart: [Build #372477](https://buildkite.com/elastic/kibana-pull-request/builds/372477) failed f660e8165e6690a261b4692d7072d0c52d7f1aac
* :broken_heart: [Build #372158](https://buildkite.com/elastic/kibana-pull-request/builds/372158) failed 1ba7cd55a9786815c6261ef6f2e3809b67c0504e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378044","number":378044,"commit":"9a6283e1ff0fdcc5e6f8a324da732dfac9024442"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372761","number":372761,"commit":"9a5d3c0f9ef99db9184528ca1e1baebb60b0bc60"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372754","number":372754,"commit":"a4fb4dccca44da47859782d6e4e5406d7ab4b5eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372732","number":372732,"commit":"c0ea163c9d5762fe5320882e7fa7662f9cb20f84"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372477","number":372477,"commit":"f660e8165e6690a261b4692d7072d0c52d7f1aac"}],"number":378044}
buildkite-pr-comment-->

tsullivan committed:
consolidate high-contrast styles into helper

tsullivan committed:
update test snapshots

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/378072)
* Commit: 0e2a0c8705bd32cb370ed20640d987f9a470ba9d

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/378072#019b8fb7-5f57-4e33-bdcb-f5efffe0749a)

### History
* :broken_heart: [Build #378044](https://buildkite.com/elastic/kibana-pull-request/builds/378044) failed 9a6283e1ff0fdcc5e6f8a324da732dfac9024442
* :broken_heart: [Build #372761](https://buildkite.com/elastic/kibana-pull-request/builds/372761) failed 9a5d3c0f9ef99db9184528ca1e1baebb60b0bc60
* :broken_heart: [Build #372754](https://buildkite.com/elastic/kibana-pull-request/builds/372754) failed a4fb4dccca44da47859782d6e4e5406d7ab4b5eb
* :broken_heart: [Build #372732](https://buildkite.com/elastic/kibana-pull-request/builds/372732) failed c0ea163c9d5762fe5320882e7fa7662f9cb20f84
* :broken_heart: [Build #372477](https://buildkite.com/elastic/kibana-pull-request/builds/372477) failed f660e8165e6690a261b4692d7072d0c52d7f1aac

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378072","number":378072,"commit":"0e2a0c8705bd32cb370ed20640d987f9a470ba9d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378044","number":378044,"commit":"9a6283e1ff0fdcc5e6f8a324da732dfac9024442"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372761","number":372761,"commit":"9a5d3c0f9ef99db9184528ca1e1baebb60b0bc60"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372754","number":372754,"commit":"a4fb4dccca44da47859782d6e4e5406d7ab4b5eb"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372732","number":372732,"commit":"c0ea163c9d5762fe5320882e7fa7662f9cb20f84"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372477","number":372477,"commit":"f660e8165e6690a261b4692d7072d0c52d7f1aac"}],"number":378072}
buildkite-pr-comment-->

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

tsullivan committed:
relocate styling code to element

kibanamachine committed:
Changes from node scripts/lint_ts_projects --fix

kibanamachine committed:
Changes from node scripts/regenerate_moon_projects.js --update

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/378110)
* Commit: 49a200004b051213c0221987be3d48b5c5c4f042
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-245898)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/49a200004b051213c0221987be3d48b5c5c4f042) | diff |
| --- | --- | --- | --- |
| `core` | 520 | 523 | +3 |
| `dashboard` | 996 | 997 | +1 |
| `developerToolbar` | 55 | 58 | +3 |
| total |  |  | +7 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/49a200004b051213c0221987be3d48b5c5c4f042) | diff |
| --- | --- | --- | --- |
| `@kbn/core-chrome-layout-utils` | 11 | 13 | +2 |
| `@kbn/core-chrome-navigation` | 11 | 12 | +1 |
| total |  |  | +3 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/49a200004b051213c0221987be3d48b5c5c4f042) | diff |
| --- | --- | --- | --- |
| `core` | 113.1KB | 115.1KB | +2.0KB |
| `fleet` | 2.1MB | 2.1MB | -169.0B |
| `observabilityOnboarding` | 126.1KB | 126.0KB | -16.0B |
| `searchHomepage` | 184.4KB | 184.3KB | -16.0B |
| `securitySolution` | 10.7MB | 10.7MB | -86.0B |
| total |  |  | +1.7KB |

#### Public APIs missing exports
> Total count of every type that is part of your API that should be exported but is not. This will cause broken links in the API documentation system. Target amount is 0. `Run node scripts/build_api_docs --plugin [yourplugin] --stats exports` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/49a200004b051213c0221987be3d48b5c5c4f042) | diff |
| --- | --- | --- | --- |
| `@kbn/core` | 918 | 919 | +1 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/49a200004b051213c0221987be3d48b5c5c4f042) | diff |
| --- | --- | --- | --- |
| `core` | 496.9KB | 497.9KB | +978.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/52a52e7360e045e41bb4d5694a9c0e66992cce76) | [after](https://github.com/elastic/kibana/commit/49a200004b051213c0221987be3d48b5c5c4f042) | diff |
| --- | --- | --- | --- |
| `@kbn/core-chrome-layout-constants` | 28 | 29 | +1 |
| `@kbn/core-chrome-layout-utils` | 30 | 40 | +10 |
| `@kbn/core-chrome-navigation` | 47 | 49 | +2 |
| total |  |  | +13 |


</details>


### History
* :broken_heart: [Build #378106](https://buildkite.com/elastic/kibana-pull-request/builds/378106) failed 7a8365ae3b9d811ec4a6fdfadfecca23d7093aa5
* :broken_heart: [Build #378072](https://buildkite.com/elastic/kibana-pull-request/builds/378072) failed 0e2a0c8705bd32cb370ed20640d987f9a470ba9d
* :broken_heart: [Build #378044](https://buildkite.com/elastic/kibana-pull-request/builds/378044) failed 9a6283e1ff0fdcc5e6f8a324da732dfac9024442
* :broken_heart: [Build #372761](https://buildkite.com/elastic/kibana-pull-request/builds/372761) failed 9a5d3c0f9ef99db9184528ca1e1baebb60b0bc60
* :broken_heart: [Build #372754](https://buildkite.com/elastic/kibana-pull-request/builds/372754) failed a4fb4dccca44da47859782d6e4e5406d7ab4b5eb

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378110","number":378110,"commit":"49a200004b051213c0221987be3d48b5c5c4f042"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378106","number":378106,"commit":"7a8365ae3b9d811ec4a6fdfadfecca23d7093aa5"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378072","number":378072,"commit":"0e2a0c8705bd32cb370ed20640d987f9a470ba9d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/378044","number":378044,"commit":"9a6283e1ff0fdcc5e6f8a324da732dfac9024442"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372761","number":372761,"commit":"9a5d3c0f9ef99db9184528ca1e1baebb60b0bc60"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/372754","number":372754,"commit":"a4fb4dccca44da47859782d6e4e5406d7ab4b5eb"}],"number":378110}
buildkite-pr-comment-->
